charts/kubernetes-external-secrets/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+name: kubernetes-external-secrets
+version: 0.1.0
+appVersion: 1.2.0
+description: Kubernetes External Secrets CustomResourceDefinition
+keywords:
+  - kubernetes-external-secrets
+  - secrets
+home: https://github.com/godaddy/kubernetes-external-secrets
```

Comment:
missing newline at the end of the file

Reply:
Damn, `helm create` generated all the source templates, will add some newlines 👍

Reply:
Eh eh, I run into this once too :D Github is pretty good to catch this :)